loggingx/logger_test.go

```diff
@@ -0,0 +1,64 @@
+package loggingx_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.infratographer.com/x/loggingx"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"go.uber.org/zap/zaptest/observer"
+)
+
+var expectedMsg = "ohno"
+
+func TestInitLogger(t *testing.T) {
+	tests := []struct {
+		name       string
+		cfg        loggingx.Config
+		expectedST bool
+	}{
+		{
+			name:       "With all disabled",
+			cfg:        loggingx.Config{Debug: false, Pretty: false, DisableStacktrace: true},
+			expectedST: false,
+		},
+		{
+			name:       "With stacktrace disabled",
+			cfg:        loggingx.Config{Debug: false, Pretty: false, DisableStacktrace: true},
+			expectedST: false,
```

Comment:
previously this test checked that the stacktrace was enabled. this should do the same, otherwise it's the same as the previous test

Reply:
fixed, thanks.